zio/bzngio/writer.go

```diff
@@ -26,12 +27,22 @@ func NewWriter(w io.Writer, flags zio.Flags) *Writer {
 	}
 }
 
+func (w *Writer) write(b []byte) (int, error) {
```

Comment:
it looks like every single caller ignores the first return value, this can return just the error?